pipeline_tools/shared/submission/create_links.py

```diff
@@ -85,11 +86,23 @@ def __init__(
         with open(output_file_path) as f:
             outputs_dict = json.load(f)
 
+        # If project level, output file will be one analysis_file
+        # Use file_name_string to get inputs
+        if project_level:
+            outputs_dict = [outputs_dict]
+
+            # input_uuids is a list, project level will be one file
+            # open first object in the array
+            with open(input_uuids[0]) as f:
```

Comment:
It looks like this iss now accepting json files (generated by the previous step) rather than UUIDs. Although I think it is possible to get it working this way in WDL, I am not sure it i the best approach, since it could be very confusing to debug

Reply:
for intermediate level runs its expecting the two fastq files as input uuids, and the outputs.json file which is the array of the metadata/analysis_file.json files.....for the project level what will be the input uuids since its not going to be the two fastq files